examples/button/index.js

```diff
@@ -8,57 +8,16 @@
  * Provided composite components like <Button /> use this to logic
  */
 
-import {getPort} from '../port';
-import ReactHardware from '../../src';
 import React, {Component} from 'react';
+import {getPort} from '../port';
+import {render, Button} from '../../src';
```

Comment:
This example intentionally had the Button in it. The intent is that when the examples source are in the documentation website having an example this would provide a full implementation of how to create a hardware component of your own—whether to implement an alternative to something in core or add support for something new.